ush/enspost_config.py

```diff
@@ -0,0 +1,215 @@
+# HRRR-E PYTHON GRIB GENERATION
+# CONFIGURATION FILE
+
+## CREATE DICTIONARIES ##
+grib_discipline = {}
+grib_category = {}
+grib_number = {}
+grib_surface = {}
+grib_level = {}
+
+##################################
+#  GENERAL ENSEMBLE SETTINGS     #
+##################################
+
+# number of members (9=RRFSE only, 10=include RRFS-B control member)
+nm = 10
+# model resolution (km)
+dx = 3
+
+# wgrib2 executable
+wgrib2 = '/home/rtrr/RAP/exec/UPP/wgrib2'
```

Comment:
Do we still need this?

Reply:
No we don't, thanks for catching that.  Removed in the latest commit